go-magic-link-example/static/serve_magic_link.html

```diff
@@ -15,7 +15,7 @@
             <div class="logged_in_div_left">
                 <div>
                     <h1>Your app,</h1>
-                    <h2>Enterprise Ready</h2>
```

Comment:
In the nav bar, could you remove the text "youre on your way" and then put the WorkOS logo on the left side? Could you also center the magic link text? 
![CleanShot 2022-08-08 at 17 18 27](https://user-images.githubusercontent.com/47123183/183530729-ab3453bd-c405-4257-9463-ad30976fce1c.png)